### PR TITLE
Normalize base URL usage in admin templates

### LINKED ISSUE #11319: Wrong relative paths for subfolder install
Via mailing list, thanks to **ejomi**:

> There are some fatal problems while running Friendica in a subfolder. I Know that this is not recommended anyway but it is possible - so why not using it.
> 
> This affects all relative links which are not dynamically/automatically adjusted by the program and there is no way to do it via redirections because they are hard coded!
> 
> You have to remove the leading slash of relative links like (example) href="/admin/addons/" into href="admin/addons/" - this problem concerns the following files:
> 
> a)
> Files with links for addons (as far as I could get):
> "ROOT/view/templates/admin/summary.tpl"
> and
> "ROOT/view/theme/frio/templates/admin/summary.tpl"
> 
> b)
> Files for the detail settings of template "frio":
> "ROOT/src/Module/Admin/Themes/Details.php"
> "ROOT/src/Module/Admin/Themes/Embed.php"
> "ROOT/src/Module/Admin/Themes/Index.php"
> (This affects the content link and return address for/from the inline frame with detail settings like color, 
> opacity etc. for the template "frio")
> 
> c)
> The file for the proper link from the main page to the detail settings of template "frio":
> "ROOT/src/Module/Admin/Site.php"
> There you have to change the link definition of language makro [$theme]
> (find text content: 'Default system theme - may be over-ridden by user...').
> 
> d)
> Changing the language markro in "Site.php" has consequences for all language files too!
> You have to adapt there both in the referring text and in the translation.
> For germany for example this would be the entry within the file "ROOT/view/lang/de/strings.php".
>  
> Hope my ideas helps a bit

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes #11319

### CODE REVIEW

File: src/Module/Admin/Themes/Index.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ protected function content(array $request = []): string
 
 		// reload active themes
 		if (!empty($_GET['action'])) {
-			self::checkFormSecurityTokenRedirectOnError(DI::baseUrl()->get() . '/admin/themes', 'admin_themes', 't');
+			self::checkFormSecurityTokenRedirectOnError('/admin/themes', 'admin_themes', 't');
```
Comment (annando) [inline src/Module/Admin/Themes/Index.php:40]: Why have you removed the base url here?

File: src/Module/Register.php [hunk 2]
```diff
@@ -380,11 +380,11 @@ protected function post(array $request = [])
 					'type'         => Model\Notification\Type::SYSTEM,
 					'event'        => 'SYSTEM_REGISTER_REQUEST',
 					'uid'          => $admin['uid'],
-					'link'         => $base_url . '/admin/users/',
+					'link'         => DI::baseUrl()->get(true) . '/admin/users/',
```
Comment (annando) [inline src/Module/Register.php:383]: why is this replaced? Doesn't `$base_url` already contain the base url?

File: src/Module/Admin/Themes/Index.php (hunk 1 above)
Comment (MrPetovan) [reply src/Module/Admin/Themes/Index.php:40]: It's applied automatically in `BaseModule::checkFormSecurityTokenRedirectOnError`, all the other calls aren't passing it for this reason. This is cleanup, only loosely related with the original issue.

File: src/Module/Register.php (hunk 2 above)
Comment (MrPetovan) [reply src/Module/Register.php:383]: We have this legacy behavior where the regular pages can be accessed with HTTP (the base URL of the node), but the admin pages are only accessed with HTTPS (self-signed certificate). In this case, `DI::baseUrl()->get()` (the value of `$baseurl`) will return the HTTP version, and `DI::baseUrl()->get(true)` will return the HTTPS version.

It's a very niche use case and we might want to remove it, but in the mean time it should be applied consistently.

**Pull request merged**